### DISCUSSION TITLE
Floating point exception (core dumped)

### DISCUSSION THREAD

**Discussion by Serana-zero** [zai-org/ChatGLM3#1236] [Q&A]
我更新了训练和推理的脚本，在transformers版本为4.41.0的情况下无法进行训练，在把transformers版本降为4.40.0的情况下，能够进行全量微调。我现在用的设备是8张H20。最后在推理的时候，报了错。
pytorch版本为2.3.0，cuda版本用的是12.1。
推理脚本用的是python inference_hf.py /home/administrator/finetune_demo/output/checkpoint-3000 --prompt 沈金康是哪里人？
<img width="941" alt="Snipaste_2024-05-26_19-34-26" src="https://github.com/THUDM/ChatGLM3/assets/51693431/f0113579-5fe3-46e7-8e52-7de6e0bba047">

**Comment by josephzpng** [comment]
hello! I also have the same problem, have you solved it?
